### DISCUSSION TITLE
Custom PDK

### DISCUSSION THREAD

**Discussion by ashkanr65** [lip6/coriolis#134] [Q&A] [answered]
Hi,

First of all, I should mention that I couldn’t find a comprehensive tutorial for using Coriolis from start to finish. However, that is not my main question. Is it possible to adopt a new PDK for Coriolis? I would like to know if it is a good idea to switch from OpenROAD to Coriolis, considering that OpenROAD has many issues.

**Comment by jpc-lip6** [comment, marked as answer]
Hello,

I agree, there is no comprehensive tutorial on Coriolis, mainly due to the lack of manpower. Did you have a look at the companion repository *alliance-check-toolkit*, despite it's name it contains lots of examples on how to use Coriolis, you may start with the ``arlet6502`` which is small and provides a lot of variants.
Of course, Coriolis can support new PDK, as long as they are above 130nm (below timing driven feature is missing). The question is not whether it can, but more *how easy it is*... In that respect, I would say it is complex, if you have an open PDK and not in a hurry I could take a look in the upcoming months. And, as each PDK has it's own peculiarities, it is a almost new work each time. Typically OpenROAD has efabless to manage that outside of the public eye.
But Coriolis has also many issues, they are just different from the ones of OpenROAD. One which is really missing is STA. We use HiTas, which is open source and good, but quite difficult to build and use.
In any case, those kind of tools needs a lot of learning to be used correctly, whether it is Coriolis or OpenROAD.

Best,
